skill-tree /src/main/java/com/RDS/skilltree/SkillTreeApplication.java

```diff
@@ -2,12 +2,20 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
 
 @SpringBootApplication
+@RestController
 public class SkillTreeApplication {
 
 	public static void main(String[] args) {
 		SpringApplication.run(SkillTreeApplication.class, args);
 	}
 
+	@GetMapping("/")
```

Comment:
Routes must go to separate controller file

Reply:
yes, that will be taken. up in the follow-up PRs this is just to have the confidence that the BE is up and running, and will be removed when we have proper APIs ready